### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by andriy-shymkiv**
- [DEMO LINK](https://andriy-shymkiv.github.io/react_decompose/)

### CODE REVIEW

**Review by Illia-Skladnik — APPROVED** [review]
Good work!

File: src/App.css [hunk 1]
```diff
@@ -1,3 +1,7 @@
+@import ".//components/Article/Article.css";
+@import ".//components/Header/Header.css";
+@import ".//components/Welcome/Welcome.css";
```
Comment (illiarerun) [inline src/App.css:3]: there's no need for this imports here
```suggestion
```

File: src/components/Article/Article.css [hunk 2]
```diff
@@ -1 +1,30 @@
...
+    margin: 3.5em auto 0;
+    padding: 2em;
+  }
+}
+
+@media (min-width: 800px) {
+  .article {
+    margin: 3.5em auto;
+  }
+}
```
Comment (illiarerun) [inline src/components/Article/Article.css:30]: put media styles right after the class that you're applying them to

File: src/components/Header/Header.jsx [hunk 3]
```diff
@@ -1,6 +1,15 @@
...
 
-// Create a Header function returning the HTML of header block
+const Header = () => (
+  <header className="header">
+    <h1 className="header__title">Site Name</h1>
+    <nav className="navigation">
+      <a className="navigation__link" href="#about">About</a>
+      <a className="navigation__link" href="#services">Services</a>
+      <a className="navigation__link" href="#contact">Contact</a>
+    </nav>
```
Comment (illiarerun) [inline src/components/Header/Header.jsx:11]: create a component for navigation

File: src/components/Header/Header.css [hunk 4]
```diff
@@ -1 +1,43 @@
...
+  font-size: inherit;
+}
+
+@media (min-width: 500px) {
+  .header {
+    text-align: left;
+    display: flex;
+    justify-content: space-between;
+  }
+}
```
Comment (illiarerun) [inline src/components/Header/Header.css:28]: put media styles right after the class that you're applying them to

**Review by illiarerun — CHANGES_REQUESTED** [review]

**Review by illiarerun — APPROVED** [review]

**Pull request closed without merging**